modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/QueryParser.java

```diff
@@ -134,6 +135,49 @@ public QueryParserResult parse(String schemaName, SqlFieldsQuery qry, boolean re
         return res;
     }
 
+    /**
+     * Create parameters from query.
+     *
+     * @param qry Query.
+     * @return Parameters.
+     */
+    public QueryParameters fromQuery(SqlFieldsQuery qry) {
```

Comment:
As this method is now a part of `QueryParser`, a name `queryParameters` for the method looks more suitable.